Capri.Services/Files/DiplomaCardCreator.cs

```diff
@@ -0,0 +1,706 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Capri.Web.ViewModels.Proposal;
+using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Wordprocessing;
+using A = DocumentFormat.OpenXml.Drawing;
+using DW = DocumentFormat.OpenXml.Drawing.Wordprocessing;
+using PIC = DocumentFormat.OpenXml.Drawing.Pictures;
+
+namespace Capri.Services.Files
+{
+    public class DiplomaCardCreator : IDiplomaCardCreator
+    {
+        private readonly string clause = "Zobowiązuję/zobowiązujemy się samodzielnie wykonać pracę w zakresie wyspecyfikowanym niżej." +
+                " Wszystkie elementy (m.in. rysunki, tabele, cytaty, programy komputerowe, urządzenia itp.)," +
+                " które zostaną wykorzystane w pracy, a nie będą mojego/naszego autorstwa będą w odpowiedni sposób" +
+                " zaznaczone i będzie podane źródło ich pochodzenia.";
+        private readonly string city = "Poznań";
```

Comment:
City and closure can be moved to some dependency. How you want later to specify language from that?